content/en/guide/v10/differences-to-react.md

```diff
@@ -142,14 +142,14 @@ function App(props) {
 
 [preact/compat] ships with specialised components that are not necessary for every app. These include
 
-- `PureComponent`: Only updates if `props` and `state` have changed
-- `memo`: Similar in spirit to `PureComponent` but allows to use a custom comparer function
-- `forwardRef`: Supply a `ref` to a specified child component.
-- `Portals`: Continues rendering the current tree into a different DOM container
-- `Suspense`: **experimental** Allows to display fallback content in case the tree is not ready
-- `lazy`: **experimental** Lazy load async code and mark a tree as ready/not ready accordingly.
+- [PureComponent](/guide/v10/switching-to-preact#purecomponent): Only updates if `props` and `state` have changed
+- [memo](/guide/v10/switching-to-preact#memo): Similar in spirit to `PureComponent` but allows to use a custom comparer function
```

Comment:
"Comparison function" seems to be used in the compat docs as well as the React docs (instead of "comparer function").